ci/scripts/e2e-kafka-sink-test.sh

```diff
@@ -150,20 +150,7 @@ echo "preparing confluent schema registry"
 python3 -m pip install --break-system-packages requests confluent-kafka
```

Comment:
But this still needs to be executed before running `python3 e2e_test/sink/kafka/register_schema.py`

Reply:
We can add a README in e2e test dir to note the required dependency